boards/posix/native_posix/doc/index.rst

```diff
@@ -537,17 +537,9 @@ The following peripherals are currently provided with this board:
 
 **Flash driver**:
   A flash driver is provided that accesses all flash data through a binary file
-  on the host file system.
-
-  The size of the flash device can be configured through the native POSIX board
-  devicetree and the sector size is configurable via the Kconfig option
-  :option:`CONFIG_FLASH_NATIVE_POSIX_SECTOR_SIZE`. The sector size will only be
-  used to return flash page layout related information and no restrictions are
-  imposed by the driver based on the configured sector size. As such an erase
-  operation of arbitrary size will succeed on the emulated flash device.
-  Further the emulated device will not impose any write restriction that are
-  applicable for a regular flash device, including changing the state of a bit
-  from zero to one.
+  on the host file system. The behaviour of the flash device can be configured
+  through the native POSIX board devicetree or Kconfig settings under
+  :option:`CONFIG_FLASH_SIMULATOR`.
```

Comment:
@Laczen As a further request: Some (rst) documentation for this would be nice.

Reply:
Actually I was thinking of removing majority of this documentation and reference the flash simulator documentation, problem is that the later one is missing. But documenting it to full extend in the native_posix documentation is definitely not the correct place.

Reply:
> Shall I create a issue to follow up this missing documentation?

it would not harm to create an "enhancement" issue :)

Reply:
done #20654